packages/app/src/components/deposit/WithdrawModal.tsx

```diff
@@ -0,0 +1,273 @@
+import { useEthers } from "@usedapp/core";
+import { useWithdraw } from "utils/tenderDepositHooks";
+import { useQuery } from "@apollo/client";
+
+type Props = {
+  show: boolean;
+  locks: Lock[];
+  protocolName: ProtocolName;
+  onDismiss: () => void;
+};
+
+const WithdrawModal: FC<Props> = ({ show, locks, protocolName, onDismiss }) => {
+  const staker = stakers[protocolName];
+  const symbol = staker.symbol;
+  const [locksUpdated, setLocksUpdated] = useState<Lock[]>([]);
+  const { library } = useEthers();
+
+  const { withdraw } = useWithdraw(protocolName);
+
+  const requiredChain = stakers[protocolName].chainId;
+  const { data: lastGovUnstakeEvent, refetch: refetchLastGovUnstakeEvent } = useQuery<Queries.LastGovernanceUnstake>(
+    Queries.GetPendingWithdrawals,
+    {
+      variables: {
+        from: getGov(protocolName).toLowerCase(),
+        tenderizer: addresses[protocolName].tenderizer.toLowerCase(),
+      },
+      context: { chainId: requiredChain },
+    }
+  );
+
+  useEffect(() => {
+    refetchLastGovUnstakeEvent();
+  }, [refetchLastGovUnstakeEvent, requiredChain]);
+
+  useEffect(() => {
+    const simulateWithdraw = async () => {
+      const updatedLocks = await Promise.all(
+        locks.map(async (lock) => {
+          const tenderizer = addresses[protocolName].tenderizer;
+          const contract = new Contract(
+            tenderizer,
+            new utils.Interface(abis.tenderizer),
+            library?.getSigner()
+          ) as Tenderizer;
+          try {
+            await contract.callStatic.withdraw(lock.unstakeLockID);
+            return {
+              ...lock,
+              open: true,
+            };
+          } catch (error) {
+            return {
+              ...lock,
+              open: false,
+            };
+          }
+        })
+      );
+      setLocksUpdated(updatedLocks);
+    };
+    simulateWithdraw();
+  }, [library, locks]);
+
+  return (
+    <>
+      {show && (
+        <Layer style={{ overflow: "auto" }} animation="fadeIn" onEsc={onDismiss} onClickOutside={onDismiss}>
+          <Card
+            flex={false}
+            pad={{ vertical: "medium", horizontal: "xlarge" }}
+            width="large"
+            style={{ position: "relative" }}
+          >
+            <Button
+              style={{ position: "absolute", top: 10, right: 10 }}
+              plain
+              icon={<FormClose />}
+              onClick={onDismiss}
+            />
+            <CardHeader justify="center" pad="none">
+              <Heading level={2} alignSelf="center">
+                {`Withdraw ${symbol}`}
+              </Heading>
+            </CardHeader>
+            <CardBody>
+              <Box pad={{ top: "medium", horizontal: "large" }} align="center">
+                <Table>
+                  <TableHeader>
+                    <TableRow>
+                      <TableCell scope="col" border="bottom">
+                        Asset
```

Comment:
this layout is nicer, otherwise you have the symbol out of place for different amounts. or you put it in a separate column after the amount but then you are back to the same thing just reversed

Reply:
But every row would just say the same thing right? Since this table would show only unstakes for a single asset

Reply:
I kind of agree with Reuben here, could consider placing the logo in front of the amount instead and outline to the left, solves the problem.